src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -89,7 +89,7 @@ public Integer extractSuffix() {
 
 	protected MutableAttributes retrieveStateAttributes() {
 		final MutableAttributes state = new MutableAttributes();
-		for ( final String attribute : AttributesProvider.getInstance().getAttributes() ) {
+		for ( final String attribute : AttributesProvider.getInstance().getJoinedAttributes() ) {
```

Comment:
This still only checks known attributes. Per default, we want _all_ (even unexpected or like for HTML5 user-defined) attributes... This then also addresses https://github.com/retest/recheck-web/issues/145